src/components/Patient/DailyRounds.tsx

```diff
@@ -900,7 +875,30 @@ export const DailyRounds = (props: any) => {
               />
             </>
           )}
-
+          <RangeAutocompleteFormField
```

Comment:
https://docs.ohc.network/docs/contributing/#how-to-resolve-merge-conflicts

Reply:
> And this shouldn't be placed here, as Sp02 will be visible for all round types, instead of just normal, telemedicine, doctor's log and community nurse's update.

@noufalrahim This is still an issue.

As mentioned earlier

> Please check the file itself, the code block that starts at [726](https://github.com/ohcnetwork/care_fe/blob/5d2d2849b6267b7e38cc57c2f4083c5b38390c2e/src/components/Patient/DailyRounds.tsx#L726) includes fields that should appear for all 4 round types (that should have SpO2 field under new requirements).
> 
> So you can move the code starting at line 848 inside that block, and just remove the code you added.

Reply:
@noufalrahim ☝️